src/sagemaker/automl/README.rst

```diff
@@ -0,0 +1,64 @@
+SageMaker Autopilot
+===================
+
+Amazon SageMaker Autopilot is an automated machine learning (commonly referred to as AutoML) solution for tabular
+datasets. It automatically trains and tunes the best machine learning models for classification or regression based
+on your data, and host a serials of models on an Inference Pipeline.
+
+SageMaker AutoML Class
+~~~~~~~~~~~~~~~~~~~~~~
+
+The SageMaker ``AutoML`` class is similar to a SageMaker ``Estimator`` where you define the attributes of an AutoML
+job and feed input data to start the job.
+
+A simple use case of an ``AutoML`` object would be:
+
+.. code:: python
+
+    from sagemaker import AutoML
+
+    auto_ml = AutoML(
+        role="sagemaker-execution-role",
+        target_attribute_name="y",
+        sagemaker_session=sagemaker_session,
+    )
+    auto_ml.fit(inputs=inputs)
+
+
+The above code will start an AutoML job (data processing, training, tuning) and output a maximum of 500 candidates by
+default. You can modify the number of output candidates by specifying ``max_candidates`` in the constructor. The AutoML
+job will figure out the problem type (BinaryClassification, MulticlassClassification, Regression), but you can also
+specify the problem type by setting ``problem_type`` in the constructor. Other configurable settings include security
+settings, time limits, job objectives, tags, etc.
```

Comment:
can we link to ReadTheDocs so people know where to find the rest of the parameters?

Reply:
I couldn't find the ReadTheDocs link for the AutoML class. For all new classes, only Analytics is there: https://sagemaker.readthedocs.io/en/stable/analytics.html

Reply:
yeah they have to be added - I've got an open PR to do so: #1161 (won't block this PR on that one, but it'd be nice to follow up on it after)